Makefile.js

```diff
@@ -306,11 +306,29 @@ function release(ciRelease) {
 function prerelease(prereleaseId) {
 
     const releaseInfo = ReleaseOps.release(prereleaseId);
+    const nextMajor = semver.inc(releaseInfo.version, "major");
+
+    /*
+     * Premajor release should have identical "next major version".
+     * Preminor and prepatch release will not.
+     * 5.0.0-alpha.0 --> next major = 5, current major = 5
+     * 4.4.0-alpha.0 --> next major = 5, current major = 4
```

Comment:
I'm a bit confused. When would this be the case? Do we even need to check for this?

Reply:
No, we don't normally do preminor and prepatch releases, but it's theoretically possible; and in those cases, the code I had before would misbehave (it would show major-release migration guide and installation information when we're not on a premajor release).